R/layout.R

```diff
@@ -162,9 +162,39 @@ config <- function(p, ..., cloud = FALSE, showSendToCloud = cloud, locale = NULL
   
   args <- list(...)
   if ("collaborate" %in% names(args)) warning("The collaborate button is no longer supported")
+  if ("modeBarButtonsToKeep" %in% names(args)) args <- modebar_keep(args)
```

Comment:
I'd prefer to make this an explicit argument in `config()` (after `...`, since it's not an official plotly.js config attribute)

Reply:
Okay. I see. I just added a [commit](https://github.com/ropensci/plotly/pull/1787/commits/ea15947839b146b850619fd3eb32f195db30a9f2) to fix this.

Reply:
Great, thanks!